src/test/java/htsjdk/variant/variantcontext/VariantContextUnitTest.java

```diff
@@ -538,6 +537,29 @@ public void testFilters() {
         Assert.assertNotNull(vc.getFiltersMaybeNull());
     }
 
+    @Test
+    public void testGenotypeFilters() {
```

Comment:
Could we rename this to testGenotypeCountsRespectGenotypeFilters or something along those lines?